src/schema/v2/fair.ts

```diff
@@ -192,6 +193,16 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
           return now.isAfter(activeStart)
         },
       },
+      isReverseImageSearchEnabled: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        description: "Have we indexed this fair's artworks to tineye?",
+        resolve: ({ id }) => {
```

Comment:
changed this from `slug` -> `id` because for some reason inside the resolver when we run the server locally (pointing to staging gravity) the slug is **ALWAYS** undefined. 🤔 

@dzucconi maybe you know why? Am I missing something? is it a historical thing that we're missing?

Reply:
Ah right. Gravity returns the slug under `id` and the internal ID under `_id`